### PR TITLE
Orca update

### PR DESCRIPTION
**Pull request by viktor-orca**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Add API Host as parameter. For support different orca sites
Limit queries to API

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by dansterenson — CHANGES_REQUESTED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -1738,9 +3920,9 @@
...
     "wordwrap": {
-      "version": "0.0.3",
-      "resolved": "https://registry.npmjs.org/wordwrap/-/wordwrap-0.0.3.tgz",
-      "integrity": "sha1-o9XabNXAvAAI03I0u68b7WMFkQc="
+      "version": "1.0.0",
+      "resolved": "https://registry.npmjs.org/wordwrap/-/wordwrap-1.0.0.tgz",
+      "integrity": "sha1-J1hIEIkUVqQXHI0CJkQa3pDLyus="
     },
     "wordwrapjs": {
       "version": "3.0.0",
```
[the remark this answers is not in the document]
Comment (viktor-orca) [reply package-lock.json]: done

**Review by dansterenson — APPROVED** [review]
Nice

File: Packs/Orca/Integrations/Orca/README.md [hunk 2]
```diff
@@ -4,19 +4,21 @@ This integration was integrated and tested with Orca
...
 
     | **Parameter** | **Description** | **Required** |
     | --- | --- | --- |
     | apikey | API Key | True |
+    | api_host | API Host without schema. Default: `api.orcasecurity.io` | False
     | first_fetch | First fetch timestamp \(`<number>` `<time unit>`, e.g., 12 hours, 7 days\) | False |
     | incidentType | Incident type | False |
     | isFetch | Fetch incidents | False |
     | max_fetch | Max fetch | False |
+    | fetch_limit | Limit number of fetched records. Default: `1000` | False |
```
Comment (dansterenson) [inline Packs/Orca/Integrations/Orca/README.md:17]: I don't see this parameter in the orca.yml file

**Review by dansterenson — CHANGES_REQUESTED** [review]

File: Packs/Orca/Integrations/Orca/README.md (hunk 2 above)
Comment (viktor-orca) [reply Packs/Orca/Integrations/Orca/README.md]: Fixed

**Review by dansterenson — APPROVED** [review]

**Pull request merged**